src/components/Tooltip/TooltipTypes.d.ts

```diff
@@ -88,13 +88,15 @@ export interface TooltipRefProps {
 export type AnchorOpenEvents = {
   mouseenter?: boolean
```

Comment:
Should we deprecate `mouseenter` and `mouseleave`?

Keeping them seems fine, but with `mouseover`/`mouseout`, would they ever really be useful?

Reply:
What will be the benefits to keep and to remove?

Reply:
I honestly can't think of any realistic situation `mouseenter`/`mouseleave` (1) would solve something that `mouseover`/`mouseout` (2) doesn't.

The only issue with (2) is that we're triggering sequential `hide`/`show` state updates on the tooltip every time the user hovers between the nested element and the parent element. I couldn't visually notice any bad behavior due to this, but it's something to keep in mind.

Maybe we just leave (1) as the default, and have (2) + a troubleshooting page for people having issues with nested anchor elements?

Reply:
Got it, about removing it: if it's not causing any bug, I would like to keep it so we support those events.

About the `contains()`, I like the idea